geomstats/backend/tensorflow/__init__.py

```diff
@@ -31,6 +31,10 @@ def get_mask_i_float(i, n):
     return mask_i_float
 
 
+def get_from_sparse(indices, data, shape):
```

Comment:
is `get_from_sparse` the name of the numpy function?

If not, does numpy have a similar function and could we use its name and syntax?

Reply:
- `get_from_sparse` is not the numpy of a numpy function
- I don't know any similar function per se, there a different ways of doing it in numpy e.g. fromfunction. My idea here would be to add this function to all backends using each specific tools, just like `to_ndarray` does

Reply:
OK! Maybe call it `sparse_array()`? Sorry I'm nit-picking, but get_from_sparse is not super intuitive?

Reply:
Yes but it's not a sparse array in the usual sense, but a dense one. Sparse refers to scipy.sparse or SparseTensor which are different data types that don't store all the zeros. The proposed method converts a sparse array to a dense (usual) one.

Reply:
Maybe `array_from_sparse`?

Reply:
Got it! `array_from_sparse` would be great: the naming is also closer to our conversion functions.